templates/dell_powerconnect_show_interfaces_description.textfsm

```diff
@@ -0,0 +1,11 @@
+Value INTERFACE (\S+)
+Value DESCRIPTION (\S.*?)
+
+Start
+  ^Port\s+Description -> Begin
+
+Begin
+  ^-+
+  ^Ch\s+Description -> End
+  ^${INTERFACE}(?:\s+${DESCRIPTION})?\s*$$ -> Record
```

Comment:
in combination with the above comment, can this just be `^${INTERFACE}\s*${DESCRIPTION}\s*$$ -> Record`

Reply:
Same comment above ;)